### ISSUE TITLE
nodeClicked don't work 

### ISSUE THREAD

**Issue by rafa355** [pb10005/diagram-vue#46]
<template>
  <section>
    <router-link to="/">Home</router-link>
    <Diagram
      :width="2000"
      :height="1000"
      :fluid="false"
      scale="1"
      background="#fafafa"
      :nodes="nodes"
      :links="links"
      :editable="true"
      :labels="{
        edit: 'Edit',
        remove: 'Remove',
        link: 'Link',
        select: 'Select',
      }"
      @editLink="editLink"
      @nodeChanged="nodeChanged"
      @nodeClicked="nodeClicked"
    ></Diagram>
  </section>
</template>
<script>
import Diagram from "diagram-vue";
import data from "../data.json";
export default {
  name: "Customise",
  components: {
    Diagram,
  },
  data() {
    return {
      editable: false,
      nodes: [],
      links: [],
    };
  },
  mounted() {
    this.nodes = data.nodes;
    this.links = data.links;
  },
  methods: {
    nodeClicked(id /* identifier of node */) {
      /* event handler */
      console.log("your clicked in node: ", id);
    },
    editLink(item) {
      console.log("aaaaa");
    },
    nodeChanged(obj) {
      this.nodes = obj.nodes;
      console.log("aqui");
    },
    linkChanged(obj) {
      this.links = obj.links;
    },
  },
};
</script>

**Comment by darlandieterich** [comment]
ok, nodeClicked don't work, and nodeChanged and editLink it's called?

**Comment by rafa355** [comment]
Yes, the functions that are not being called are nodeClicked and linkClicked.

**Comment by darlandieterich** [comment]
what version your using? here work fine!
